conductor/src/main.rs

```diff
@@ -368,29 +368,29 @@ async fn run(metrics: CustomMetrics) -> Result<(), ConductorError> {
 
                 let result = get_one(client.clone(), &namespace).await;
 
-                let extension_still_processing = match &result {
-                    Ok(coredb) => extensions_still_processing(coredb),
-                    Err(_) => true,
-                };
-
-                if extension_still_processing
-                    && (read_msg.message.event_type == Event::Create
-                        || read_msg.message.event_type == Event::Restore)
-                {
-                    let _ = queue
-                        .set_vt::<CRUDevent>(
-                            &control_plane_events_queue,
-                            read_msg.msg_id,
-                            REQUEUE_VT_SEC_SHORT,
-                        )
-                        .await?;
-                    metrics.conductor_requeues.add(
-                        &opentelemetry::Context::current(),
-                        1,
-                        &[KeyValue::new("queue_duration", "short")],
-                    );
-                    continue;
-                }
+                // let extension_still_processing = match &result {
```

Comment:
Can you please just delete instead of commenting out

Reply:
oops yeah, thats my bad.